### DISCUSSION TITLE
Can I use ESP32-PICO-MINI-02 support for Parallel 1P mode

### DISCUSSION THREAD

**Discussion by olderzeus** [darthcloud/BlueRetro#1052] [Q&A]
I would like to use an ESP32-PICO-MINI-02 to connect to DS Lite in Parallel 1P mode, but some GPIO pins doesn‘t match. Can I modify the GPIO definitions in the BlueRetro/main/adapter/wire/paralleliz_1p.c and compile it to take effect? Can automatic adaptation be achieved through BlueRetro.io configuration?
Request develop a specific Parallel 1P custom adaptation mode, which can be configured by the user based on GPIO, and more button configurations can be introduced according to different ESP32 modules.

**Comment by darthcloud** [comment]
I might do it in the future. Not sure when tho.
